### PR TITLE
Pam 232 platform cleanup

### PR DESCRIPTION
**Pull request by pamgoodrich**
Relates to #232.

### CODE REVIEW

File: 2.-workspaces/configure-git/d.github-enterprise.md [hunk 1]
```diff
@@ -25,8 +25,8 @@ Read more about [creating a GitHub OAuth App](https://developer.github.com/apps/
 2. Select **Add** for GitHub.
 3. Select **Customize integration settings**.
 4. Add the following information:
-   - **Host URL**: This is the URL where your instance of GitHub Enterprise is hosted.
-   - **API URL**: This is the URL of the API of your instance of GitHub Enterprise. This is usually: `api.{HostURL}`.
+   - **Host URL**: Enter `https://github.com`.
+   - **API URL**: Enter `https://api.github.com`.
```
Comment (chohmann) [inline 2.-workspaces/configure-git/d.github-enterprise.md:29]: 🤔  hmmm so this is correct for GitHub Cloud, so basically their SAAS offering. If this guide is supposed to instruct how to setup a self-hosted version of GitHub then the previous statement would be correct. I'm not 100% sure GitHub offers an on-prem/self-hosted version or not.

Also for the major SAAS providers, we've already have default configurations for the integrations that people can just use:

image

Maybe we should call that out somewhere?

File: 2.-workspaces/configure-git/e.gitlab.md [hunk 2]
```diff
@@ -24,14 +24,15 @@ Read more about [creating a GitLab OAuth App](https://docs.gitlab.com/ee/integra
 1. Select **Settings** from the workspace home page, then navigate to the **Integrations** section.
 2. Select **Add** for GitLab.
 3. Select **Customize integration settings**, and then add the following information:
-   - **Host URL**: This is the URL where your instance of GitLab is hosted.
-   - **API URL**: This is the URL of the API of your instance of GitLab. This is usually: `{HostURL}/api/v4`
+   - **Host URL**: Enter `https://gitlab.com`.
+   - **API URL**: Enter `https://gitlab.com/api/v4`.
```
Comment (chohmann) [inline 2.-workspaces/configure-git/e.gitlab.md:28]: I know for a fact you can have a self-hosted/on prem version of GitLab, because we have one ourselves for testing purposes: https://gitlab.infra.stoplight-dev.com/.

So the previous statement is correct for configuring Stopligt for a self-hosted/on-prem version of GitLab.

Again it may not be helpful to call out how to configure these integrations for the SASS offerings, because we already have default configurations for that that every workspace can use. 

I think we only really need to help people who have self-hosted/on-prem instances of these integrations.

**Review by chohmann — COMMENTED** [review]
I'm not really sure how to go about this...

Overall, we have default configurations for the SASS offerings for Bitbucket Cloud, GitHub, GitLab. So we really don't need to tell people how to configure those. They can just choose the `Use default configuration` option when installing the integrations into their workspace.

image

So it may make sense to have this section of the docs only talk about how to configure self-hosted/on-prem instances of these integrations. 🤷‍♀️  We'd have to do some research to see which integrations even have a self-hosted option, and if they do confirm what the URLs should be for each.

Or maybe we just call out what you should do for both self-hosted and On prem in the Host URL and API URL part?

File: 2.-workspaces/configure-git/d.github-enterprise.md (hunk 1 above)
Comment (pamgoodrich) [reply 2.-workspaces/configure-git/d.github-enterprise.md]: Done

File: 2.-workspaces/configure-git/e.gitlab.md (hunk 2 above)
Comment (pamgoodrich) [reply 2.-workspaces/configure-git/e.gitlab.md]: Done

File: 2.-workspaces/configure-git/b.bitbucket-cloud.md [hunk 3]
```diff
@@ -28,16 +28,13 @@ Read more about [creating a BitBucket OAuth consumer](https://support.atlassian.
 2. Select **Add** for Bitbucket Cloud.
 3. Select **Customize integration settings**.
 4. Add the following information:
-   - **Host URL**: This is the URL where your instance of Bitbucket Cloud is hosted.
-   - **API URL**: This is the URL of the API of your instance of Bitbucket Cloud. This is usually: ***api.{HostURL}/2.0***
+   - **Host URL**: Enter `https://bitbucket.org`.
+   - **API URL**: Enter `https://api.bitbucket.org/2.0`.
```
Comment (chohmann) [inline 2.-workspaces/configure-git/b.bitbucket-cloud.md:32]: I think we should revert this change as well. Now that we are making this doc describe self-hosted bitbucket cloud instance (regardless if that functionality actually exists or not) `api.{HostURL}/2.0` is most likely correct for self-hosted.

**Review by chohmann — APPROVED** [review]
Just one little thing, but other than that 👍  Giving my approval in case I'm out by the time you make the change.
